tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -106,7 +107,8 @@ def setUp(self):
         # We're redefining project to ensure that the individual tests have unique project ids.
         # Sharing project ids across tests could result in some race conditions
         self.project = self.create_project()
-        self.integration = install_slack(self.organization)
+        self._integration = install_slack(self.organization)
+        self.integration = integration_service._serialize_integration(self._integration)
```

Comment:
The methods in the test were being invoked with the django orm integration, but in prod they were using APIIntegrations. Changing this passes in the APIIntegration. 

If just this change was made, without the changes in each of the unfurl files, this test lights up red, so if CI passes that means this fix is working :)

Reply:
@leeandher unrelated to this PR. But would we need to update `self.integration` in a lot of the integration tests to make use of `integration_service._serialize_integration()`?